styles.css

```diff
@@ -1,21 +1,75 @@
 body {
-  max-width: 800px;
   margin: 0 auto;
+  color:darkblue;
 }
 
 h1 {
-  color: blue;
   font-size: 4em;
   text-align: center;
+  font-style: italic;
+  padding: 1em;
 }
 
-.heading-item {
-  list-style-type: none;
-  font-weight: bold;
-  margin: .5em auto;
+form {
+  position: relative;
+  text-align: center;
 }
 
 #countdownDiv {
   text-align: center;
   font-size: 4em;
 }
+
+.background-image {
+  position: fixed;
+  left: 0;
+  top: 0;
+  z-index: 1;
+  background-image: url("img/img5.jpg");
+  width: 100%;
+  height: 100%;
+  
+  background-attachment: scroll;
+  background-repeat: no-repeat;
+  background-size: cover;
+  -webkit-filter: blur(5px);
+  -moz-filter: blur(5px);
+  -o-filter: blur(5px);
+  -ms-filter: blur(5px);
+  filter: blur(5px);
+}
+
+.content {
+  position: relative;
+  left: 0;
+  top: 0;
+  z-index: 9999;
```

Comment:
you can place just the number 2, it is relative to the one above, so it doesn't need to be so big.

Reply:
Yes, sure, it was just 9999 here to show "the topmost layer". It's usually done like that with some kind of a "big number" in order not to change it, when something would be added in between (like for extensibility reason).
But you're completely right, in the code we have there is no difference of having 9999 or 2 here.